checks/mysql

```diff
@@ -250,3 +248,171 @@ check_info['mysql.connections'] = {
     "group": "mysql_connections",
     "has_perfdata": True,
 }
+
+#.
+#   .--Galera Sync Status--------------------------------------------------.
+#   |         ____       _                  ____                           |
+#   |        / ___| __ _| | ___ _ __ __ _  / ___| _   _ _ __   ___         |
+#   |       | |  _ / _` | |/ _ \ '__/ _` | \___ \| | | | '_ \ / __|        |
+#   |       | |_| | (_| | |  __/ | | (_| |  ___) | |_| | | | | (__         |
+#   |        \____|\__,_|_|\___|_|  \__,_| |____/ \__, |_| |_|\___|        |
+#   |                                             |___/                    |
+#   |                    ____  _        _                                  |
+#   |                   / ___|| |_ __ _| |_ _   _ ___                      |
+#   |                   \___ \| __/ _` | __| | | / __|                     |
+#   |                    ___) | || (_| | |_| |_| \__ \                     |
+#   |                   |____/ \__\__,_|\__|\__,_|___/                     |
+#   |                                                                      |
+#   +----------------------------------------------------------------------+
+
+def inventory_mysql_galerasync(parsed):
+    for instance, values in parsed.iteritems():
+        if values.get('wsrep_provider', 'none') != 'none' and u'wsrep_local_state_comment' in values:
```

Comment:
values.get('KEY') is not None
1. 'get' returns None if key not found
2. use 'is' operator for checking agains object's identity ('==' to check equality)

Reply:
I think this has been written that way because wsrep_provider may also return False if set and then the if does not eval correctly.

Reply:
Sure it does:
`>>> d = {}
>>> d['k'] = False
>>> d.get('k', 'none') != 'none'
True
>>> d.get('k') is not None
True
>>> d.get('l', 'none') != 'none'
False
>>> d.get('l') is not None
False`